entrypoint.sh

```diff
@@ -31,6 +38,7 @@ headers=$(https --ignore-stdin --check-status --headers \
   $hubhost/api/test-migrations \
   "Authorization: Bearer $FLYWAY_HUB_ACCESS_TOKEN" \
   projectId:=$projectId \
+  flywayConf:=$flywayConfPath \
```

Comment:
I don't think you need the semicolon here: https://httpie.io/docs#non-string-json-fields

You want this path passed as a quoted string to the api right?

Reply:
When the flyway conf is not provided, I want to pass the raw json `null` value so as to inform the API it should use its existing project config. When it is, I want to use the value as a string.

`:=` appears to use "raw" json values so these semantics are correct AFAICT.